CovertInfiltration/Src/CovertInfiltration/Classes/X2StrategyElement_DefaultComplications.uc

```diff
@@ -71,12 +71,18 @@ function SpawnRescueMission(XComGameState NewGameState, XComGameState_ActivityCh
 
 			if (ItemState != none)
 			{
-				NewChainState.ChainObjectRefs.AddItem(ChainState.ChainObjectRefs[i]);
+				Package.ItemType = ItemState.GetMyTemplateName();
+				Package.ItemAmount = ItemState.Quantity;
+				ResContainer.Packages.AddItem(Package);
 				// TODO: in spawned chain, make the rewards reflect the saved items
 			}
 		}
 	}
-
+	
+	NewChainState = ChainTemplate.CreateInstanceFromTemplate(NewGameState);
+	NewChainState.FactionRef = ChainState.FactionRef;
```

Comment:
There is a bit of confusion between `ChainState` and `NewChainState`... I propose following names: `InterceptedChainState` and `SpawnedChainState`